packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -141,15 +126,6 @@ AccordionItem.propTypes = {
    */
   open: PropTypes.bool,
 
-  /**
-   * The callback function to render the expand button.
-   * Can be a React component class.
-   */
-  renderExpando: deprecate(
```

Comment:
I thought this was one of the props that we deprecated as part of v11? I recall Ale and Scott working on this, I think.

Reply:
Good catch. I'll roll these back.